src/blocks/Section/ComponentClient.tsx

```
@@ -0,0 +1,34 @@
+
+  const mergedStyle = {
+    ...(backgroundColor ? { backgroundColor: backgroundColor } : {}),
+    ...(padding ? { padding: padding } : {}),
+    ...style,
+  }
+
+  return (
+    <Section style={mergedStyle}>
+      {content && Array.isArray(content) && content.length > 0
+        ? content.map((block: Block) =>
+            BlockRendererClient({ block, previewMode, imageCollectionSlug }),
```

ESLint

[error] Expected "imageCollectionSlug" to come before "previewMode".

(perfectionist/sort-objects)